src/libraries/System.Net.HttpListener/src/System/Net/Managed/HttpConnection.cs

```diff
@@ -79,18 +81,22 @@ public HttpConnection(Socket sock, HttpEndPointListener epl, bool secure, X509Ce
             _cert = cert;
             if (secure == false)
             {
+#pragma warning disable CA1416 // Validate platform compatibility
                 _stream = new NetworkStream(sock, false);
```

Comment:
Called in Unix, Browser context, System.Net.Sockets  assembly is unsupported, seems NetworkStream is supported on browser, this type accesses Socket a lot, or maybe this type bettor to be excluded from browser context

Reply:
>  maybe this type bettor to be excluded from browser context

From my initial take, I think this is the right option.

---
In reply to: [504142071](https://github.com/dotnet/runtime/pull/43363#discussion_r504142071) [](ancestors = 504142071)

Reply:
To make sure I understand correctly, is the proposal to apply `UnsupportedOSPlatform("browser")]` to the `HttpConnection` class?

Reply:
By `to be excluded from browser context` i meant not include this type for browser build, anyways with https://github.com/dotnet/runtime/pull/43401 this assembly going to be annotated with Unsupported browser, so this and all other suppression in `System.Net.HttpListener` will not needed anymore